internal/pgbackrest/postgres.go

```diff
@@ -39,7 +39,8 @@ func PostgreSQL(
 	outParameters.Mandatory.Add("archive_mode", "on")
 
 	if backupsEnabled {
-		outParameters.Mandatory.Add("archive_command", archive)
+		// outParameters.Mandatory.Add("archive_command", archive)
+		outParameters.Mandatory.Add("archive_command", `true`)
```

Comment:
@senyosimpson I think this is not doing what we want it to do?

By setting the "archive_command" to `true` instead of`pgbackrest --stanza... archive-push "%p"`, we won't run the pgbackrest archive-push command and will miss WAL archiving altogether, which is important for restores / PITR, etc.